Backpack/Toast/Classes/BPKToast.m

```diff
@@ -128,12 +131,19 @@ - (void)hudWasHidden:(MBProgressHUD *)hud {
     }
 }
 
+#pragma mark - Private methods
+
 - (void)setupHUD {
     self.hud.bezelView.style = MBProgressHUDBackgroundStyleSolidColor;
     self.hud.bezelView.backgroundColor = [[BPKColor dynamicColorWithLightVariant:BPKColor.skyGray darkVariant:BPKColor.blackTint03] colorWithAlphaComponent:0.85];
     self.hud.contentColor = BPKColor.white;
     self.hud.delegate = self;
     self.hud.accessibilityIdentifier = ToastAccessibilityIdentifier;
+    self.hud.accessibilityViewIsModal = YES;
+}
+
+- (void)postShowAccessibilityNotification {
```

Comment:
Maybe the name of this method could be clearer? Maybe `postAccessibilityNotification`?

Reply:
Sure